website/landingpage/dev/gulpfile.babel.js

```diff
@@ -174,7 +178,7 @@ function reload(done) {
 }
 
 function buildBlogFiles(done) {
-  writeBlogFiles();
+  processBlogFiles();
```

Comment:
I think 'buildBlogFiles' needs to be added to the default gulp task so that it can be tested/styled locally. 
For now 'npm run start' doesn't generate a new blog entry on the web site. 

What do you think, @maxmarkus ?